code/Attack.ipynb

```diff
@@ -0,0 +1,1620 @@
+{
```

Comment:
<p>We don't know anything about the dataset. Provide a bride description. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>